website/src/mdRoutes.js

```diff
@@ -1,65 +1,63 @@
-import presentation from '../../docs/presentation.md';
```

Comment:
i was trying to figure out where to put highlight and hexbin and I rage quit trying to mentally diff the whole thing